Libraries/CMSIS/Device/Maxim/MAX32657/Source/GCC/max32657.ld

```diff
@@ -61,19 +61,20 @@ SECTIONS {
         _etext = .;
     } > FLASH_S
 
-    .ARM.extab :
+    /*
+     * Secure Gatway (SG) veneers.
+     *  All SG veneers are placed in the special output section .gnu.sgstubs.
+     */
+    .gnu.sgstubs :
     {
-        *(.ARM.extab* .gnu.linkonce.armextab.*)
+        _sg_veneers = .;
+        KEEP(*(.gnu.sgstubs*))
+        _esg_veneers = .;
     } > FLASH_S
```

Comment:
See https://github.com/analogdevicesinc/msdk/commit/4e9156e999a23ed16cbaa7125005362013771573

I updated the linkerfile to support secure/non-secure from the same file.  I think we should be able to wrap this section in the same way so that it only gets generated for the secure builds.